resources/js/Pages/Home.vue

```diff
@@ -135,12 +133,8 @@
 <style lang="scss">
 @import '~@wmde/wikit-tokens/dist/_variables.scss';
 
-#about-description {
+#intro-section {
```

Comment:
Nice retargeting! In order to reuse this rule for both sections I would do something like this:

```suggestion
#intro-section,
#message-section {
```

Reply:
Do I leave the border-radius of .wikit-Message inside this shared rule then?
```
#about-description,
#message-section {
    max-width: 705px;

    .wikit-Message {
        border-radius: $border-radius-base;
}
```
or only share the 705px and define the radius alone, on the top level?
```
#about-description,
#message-section {
    max-width: 705px;
}

.wikit-Message {
    border-radius: $border-radius-base;
}
```

Reply:
second option please :)

Reply:
cool, thanks!